code/index.html

```diff
@@ -1,30 +1,276 @@
 <html>
   <head>
+    <script src="https://use.fontawesome.com/bb61c0507d.js"></script>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Lizard</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/css/main.css" />
+    <meta name="description" content="A cool company for workout" />
   </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
+  <body onscroll="transformSection()">
+    <!-- START HEADER -->
+    <header class="main-header">
+      <video autoplay muted loop class="header-video">
+        <!-- <source src="./assets/video/01.mp4" type="video/mp4" />
+        <source src="./assets/video/01.webm" type="video/webm" /> -->
 
-    <!-- Signup form -->
+        <source
+          src="./assets/video/SampleVideo_1280x720_1mb.mp4"
+          type="video/mp4"
+        />
+        <source
+          src="./assets/video/SampleVideo_1280x720_1mb.webm"
+          type="video/webm"
+        />
 
+        <source
+          src="./assets/video/SampleVideo_720x480_1mb.mp4"
+          type="video/mp4"
+          media="(max-width: 900px)"
+        />
+        <source
+          src="./assets/video/SampleVideo_720x480_1mb.webm"
+          type="video/webm"
+          media="(max-width: 900px)"
+        />
+
+        <source
+          src="./assets/video/SampleVideo_360x240_1mb.mp4"
+          type="video/mp4"
+          media="(max-width: 600px)"
+        />
+        <source
+          src="./assets/video/SampleVideo_360x240_1mb.webm"
+          type="video/webm"
+          media="(max-width: 600px)"
+        />
+
+        <p>This browser doesn't support video...</p>
+      </video>
+      <!-- LOGO -->
+      <a href="/" class="block logo"
+        ><img src="./assets/img/logo.png" alt=""
+      /></a>
+      <a href="#" class="menu"><i class="fa fa-bars"></i></a>
+
+      <h1 class="main-title">
+        <blockquote>
+          Every mountain top is within reach if you just keep climbing.
+        </blockquote>
+      </h1>
+      <!-- SIGN UP -->
+      <a href="./pages/form.html" class="btn-container">
+        <button type="submit" class="signup-btn">Join us</button></a
+      >
+    </header>
+    <!-- END OF HEADER -->
+
+    <!-- START FIRST BOX -->
+    <div class="main-wrapper">
+      <section id="first-box" class="first-box">
+        <header class="first-box-title">
+          <h2>
+            Explore more about climbing and how you can be a professional
+            climber.
+          </h2>
+        </header>
+        <ul class="first-box-list">
+          <li class="first-box-items">
+            <a href="#" class="block">
+              <figure>
+                <img src="./assets/img/02.webp" alt="" class="res-img" />
```

Comment:
There is really no point in setting `alt` attribute as empty string, let's add some meaningful message there